ci/run_premerge_cpu.sh

```diff
@@ -52,21 +52,26 @@ verify_bundle() {
             echo $bundle_list
             for bundle in $bundle_list;
             do
-                pip install -r requirements-dev.txt
-                # get required libraries according to the bundle's metadata file
-                requirements=$(python $(pwd)/ci/get_bundle_requirements.py --b "$bundle")
-                # check if ALLOW_MONAI_RC is set to 1, if so, append --pre to the pip install command
-                if [ $ALLOW_MONAI_RC = true ]; then
-                    include_pre_release="--pre"
+                if [ "$bundle" != "maisi_ct_generative" ]; then
```

Comment:
We didn't have a formal way to skip verify bundle right?

Reply:
added a list to contain it